packages/sdk-v2/src/sdk/goodid/db/createDb.ts

```diff
@@ -0,0 +1,17 @@
+import Dexie from "dexie";
+
+/**
+ * Schema definition for the credentials collection
+ * - `type`: should indicate the type of the credential
+ * - Other fields are according to Veramo's Credential schema
+ */
+
+const schema = {
+  credentials: ",type, credentialSubject.id, issuer.id, issuanceDate"
```

Comment:
you'd need 2 tables as we discussed

**certificates (dictionary):** id, issuer, date, json
 **certificates_credentials (transitive table):** certificate_id, credential (ENUM of https://github.com/GoodDollar/GoodServer/blob/master/src/server/goodid/goodid-middleware.js#L126)

(1) certificates.id => (∞) certificates_credentials.certificate_id

query will be like `SELECT certificates.*, certificates_credentials.credential FROM certificates_credentials INNER JOIN certificates ON certificates_credentials.certificate_id = certificates.id` (written in methods call chain ofc)

Reply:
After looking at it for a minute I am not sure this relation makes sense.
A certificate in our definition is a collection of credentials.
Who gives out a certificate? an issuer (which has a unique ID).

So should the relationship not be: Issuer > Certificates > Collection of credentials.
Ending up with:
```
issuers: id (issuers_id), name
certificates: ++index, credentialSubject_id, issuer_id, type, issuanceDate
```
To then for example:
```
SELECT certificates.*
FROM certificates
INNER JOIN issuers ON certificates.issuer_id = issuers.id
WHERE issuers.name = 'GoodDollar';
```

Reply:
The name added is for clarity purposes. it could be done with just the issuer id I guess

Reply:
Added here: https://github.com/GoodDollar/GoodWeb3-Mono/pull/128/commits/0d1bb391bcbd42a1fa00099122c870a97f35b352